addons/sys_list/stringtable.xml

```diff
@@ -35,73 +35,85 @@
             <German>Zeigt Benachrichtigungen für Funk und Funkgeräte.</German>
             <Japanese>無線機と送信に関する通知を表示します。</Japanese>
+            <Chinese>顯示哪個無線電通訊通知</Chinese>
             <Russian>Показывает уведомления с рациями и переговорами</Russian>
         </Key>
         <Key ID="STR_ACRE_sys_list_HintBackgroundColor_DisplayName">
             <English>Notification Background Color</English>
             <German>Hintergrundfarbe der visuellen Benachrichtigung</German>
             <Russian>Цвет фона уведомления</Russian>
             <Japanese>通知の背景カラー</Japanese>
+            <Chinese>通知的背景顏色</Chinese>
         </Key>
         <Key ID="STR_ACRE_sys_list_HintTextFont_DisplayName">
             <English>Notification Text Font</English>
             <German>Schriftart der visuellen Benachrichtigung</German>
             <Russian>Шрифт текста уведомлений</Russian>
             <Japanese>通知テキストのフォント</Japanese>
+            <Chinese>通知的字型</Chinese>
         </Key>
         <Key ID="STR_ACRE_sys_list_LanguageColor_DisplayName">
             <English>Babel Cycle Language Color</English>
             <German>Schriftfarbe während Babel Sprachwechsel</German>
             <Russian>Цвет при изменении языка Babel</Russian>
             <Japanese>バベル サイクル言語カラー</Japanese>
+            <Chinese>多國語言切換顏色</Chinese>
         </Key>
         <Key ID="STR_ACRE_sys_list_SwitchChannelColor_DisplayName">
             <English>Switch Channel Color</English>
             <German>Farbe bei Funkkanalwechsel</German>
             <Russian>Цвет при смене каналов</Russian>
             <Japanese>チャンネル切り替えカラー</Japanese>
+            <Chinese>切換頻道顏色</Chinese>
         </Key>
         <Key ID="STR_ACRE_sys_list_ToggleHeadsetColor_DisplayName">
             <English>Toggle Headset Color</English>
             <German>Farbe bei Auf- oder Absetzen der Kopfhörer</German>
             <Russian>Изменить цвет наушников</Russian>
             <Japanese>ヘッドセット切り替えカラー</Japanese>
+            <Chinese>耳機切換顏色</Chinese>
         </Key>
         <Key ID="STR_ACRE_sys_list_CycleRadiosColor_DisplayName">
             <English>Cycle Radio Color</English>
             <German>Farbe bei Funkgerätwechsel</German>
             <Russian>Цвет при смене рации</Russian>
             <Japanese>無線機切り替えカラー</Japanese>
+            <Chinese>無線電循環顏色</Chinese>
         </Key>
         <Key ID="STR_ACRE_sys_list_PTT1Color_DisplayName">
             <English>Radio 1 (PTT 1) Transmit Color</English>
             <German>Farbe Sprechtaste 1</German>
             <Russian>Рация 1 (PTT 1) цвет передачи</Russian>
             <Japanese>無線機 1 (PTT 1) 送信カラー</Japanese>
+            <Chinese>無線電1 (隨按即說1)通訊顏色</Chinese>
         </Key>
         <Key ID="STR_ACRE_sys_list_PTT2Color_DisplayName">
             <English>Radio 2 (PTT 2) Transmit Color</English>
             <German>Farbe Sprechtaste 2</German>
             <Russian>Рация 2 (PTT 1) цвет передачи</Russian>
```

Comment:
This is also wrong, can you fix it as well?

Reply:
ah,you mean which line?

Reply:
oh,i saw it